### DISCUSSION TITLE
Is there anyway to turn off cmp-completion or null-ls linter if they are already set?

### DISCUSSION THREAD

**Discussion by qvieth** [LunarVim/LunarVim#2033] [Q&A] [answered]
For example, I'm opening a file, and completion and linter are already activated. Is there any vim script command so that I can turn off these things(or turn these things on later)?

**Comment by abzcoding** [comment, marked as answer]
changing the loading of plugins is in our roadmap as mentioned in https://github.com/LunarVim/LunarVim/pull/2300#issuecomment-1048602620

regarding the changes yes you have couple of routes, you can either change `lvim.builtin.cmp.sources` or use autocommands to enable/disable sources based on filetype
